android/src/main/java/com/onegini/mobile/sdk/reactnative/RNOneginiSdk.kt

```diff
@@ -104,57 +106,71 @@ class RNOneginiSdk(reactContext: ReactApplicationContext) : ReactContextBaseJava
 
     @ReactMethod
     fun getAllAuthenticators(profileId: String?, promise: Promise) {
-        profileId ?: promise.rejectWithNullError("profileId", "String").run { return }
-        sdkWrapper.getAllAuthenticators(profileId, promise)
+        when (profileId) {
+            null -> promise.rejectWithNullError("profileId", "String")
+            else -> sdkWrapper.getAllAuthenticators(profileId, promise)
+        }
     }
 
     @ReactMethod
     fun getRegisteredAuthenticators(profileId: String?, promise: Promise) {
-        profileId ?: promise.rejectWithNullError("profileId", "String").run { return }
-        sdkWrapper.getRegisteredAuthenticators(profileId, promise)
+        when (profileId) {
+            null -> promise.rejectWithNullError("profileId", "String")
```

Comment:
All of the `keys` that are repeated throughout the class like `prifleId` & `pin` can be extracted into `private const`val` in `companion object` of this class.